build.gradle

```
@@ -84,7 +84,8 @@ allprojects {
                         'Java8ApiChecker', // tritium requires JDK 11+
                         'MemberName', // false positives on ignored lambda args
                         'StaticOrDefaultInterfaceMethod', // Android specific
-                        'Var' // high noise, low signal
+                        'Var', // high noise, low signal
+                        'Varifier' // don't `var`ify everything yet
```

Handling this in baseline as well: https://github.com/palantir/gradle-baseline/pull/2181

---

yeah, `VarUsage` in gradle-baseline currently throws `error: [VarUsage] The var keyword results in illegible code in most cases and should not be used.` so conflicts with `Varifier`
```suggestion
                        'Varifier' // don't `var`ify everything yet as this conflicts with baseline VarUsage
```